### PR TITLE
adding my artsyness to the project

### PR DESCRIPTION
**Pull request by nisan137**
Please add me to this list. @royranger

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -5759,8 +5759,15 @@ let cards = [
     imageLink: './Art/letz/trans.gif',
     author: 'letz0703',
     githubLink: 'https://github.com/letz0703'
+  },
+  {
```
Comment (donmarvex) [inline include.js:5763]: Please add the artName property and value

File: include.js [hunk 2]
```diff
@@ -5759,8 +5759,15 @@ let cards = [
...
+    pageLink: './Art/Nisanim/index.html',
+    imageLink: './Art/Nisanim/Anis.gif',
+    author: 'Nisan',
+    githubLink: 'https://github.com/nisan137'
   }
 ];
+];
```
Comment (donmarvex) [inline include.js:5770]: Also remove extra closing square bracket

**Review by donmarvex — CHANGES_REQUESTED** [review]
Hey Anis, please check these changes and resolve the merge conflicts.

**Comment by donmarvex** [comment]
Hello @nisan137 , you have not made the requested changes to your PR.

**Comment by nisan137** [comment]
hii i did the changes


‫בתאריך יום ג׳, 10 במאי 2022 ב-15:06 מאת ‪Marvelous Emoghene‬‏ <‪
***@***.***‬‏>:‬

**Comment by nisan137** [comment]
Ok I will do it soon :)

בתאריך יום ג׳, 10 במאי 2022, 16:37, מאת Marvelous Emoghene ‏<
***@***.***>:

**Pull request closed without merging**